plugins/fused-ops-and-kernels/src/fms_acceleration_foak/models/mixtral.py

```diff
@@ -93,6 +95,11 @@ def get_mp_rules(base_type):
                 "transformers.models.mixtral.modeling_mixtral",
             ),
         ),
+        ModelPatcherRule(
```

Comment:
the better way is to do a swtich between the two rules `mixtral-cross-ent"` and `mixtral-fused-lce`, because they shouldnt be installed simultanosuly. If we know that we want fused, then only the fused rule should be installed.